experimental/bridge/opentracing/internal/mock.go

```diff
@@ -0,0 +1,333 @@
+package internal
```

Comment:
I'm curious whether https://github.com/opentracing/opentracing-go/tree/master/mocktracer doesn't support your use? If not, maybe add a comment.

Reply:
This mock tracer implements OpenTelemetry API. The one under the link implements the OpenTracing API.